src/constants.js

```diff
@@ -0,0 +1,12 @@
+export const REDUX_DEVTOOLS_EXTENSION = '__REDUX_DEVTOOLS_EXTENSION__';
+
+export const SORTING_BY_RATING = Symbol('RATING');
```

Comment:
I think there are no reasons for using Symbol here. The constant strings will be enough.

Reply:
done